discojs/src/dataset/data_loader/data_loader.ts

```diff
@@ -1,12 +1,17 @@
 import { Dataset } from '../dataset_builder'
 import { Task } from '../../task'
 
-export interface DataConfig { features?: string[], labels?: string[], shuffle?: boolean }
+export interface DataConfig {features?: string[], labels?: string[], shuffle?: boolean, valSplit?: number}
 export interface Data {
   dataset: Dataset
   size: number
 }
 
+export interface DataTuple {
+  train: Data
+  val: Data
```

Comment:
the validation set should be optional, no? btw i think giving it the full name "validation" is fine